classes/MediaSubfolder.js

```diff
@@ -10,10 +10,17 @@ class MediaSubfolder {
     switch (fileType) {
       case 'images':
         this.fileType = new ImageType()
+        this.mediaFolderName = fileType
+        break
+      case 'documents':
+        this.fileType = new DocumentType()
+        this.mediaFolderName = 'files'
+        break
       default:
         this.fileType = new ImageType()
+        this.mediaFolderName = fileType
+        break
```

Comment:
nit: is there a reason for defaulting to `ImageType`? Maybe we could throw an error or not have a default case, as this feels like this leaves an area for misuse that might cause downstream errors.

Reply:
Good point, I've made it throw an error instead for an unknown type - but have left it as a generic error, since this is mostly for our debugging purposes - https://github.com/isomerpages/isomercms-backend/pull/169/commits/0ae5456007b0866195e7ba2ba16e44e830f3e477